aws/biganimal-csp-setup

```diff
@@ -239,7 +239,7 @@ create_iam_policy()
*","eks:List*","eks:TagResource","eks:UntagResource","eks:UpdateAddon","eks:UpdateNodegroupConfig","iam:AttachRolePolicy","iam:CreateOpenIDConnectProvider","iam:CreatePolicy","iam:CreateRole","iam:DeleteOpenIDConnectProvider","iam:DeletePolicy","iam:DeleteRole","iam:DeleteRolePolicy","iam:DetachRolePolicy","iam:GetAccountSummary","iam:GetOpenIDConnectProvider","iam:GetPolicy","iam:GetPolicyVersion","iam:GetRole","iam:GetRolePolicy","iam:ListAccessKeys","iam:ListAttachedRolePolicies","iam:ListEntitiesForPolicy","iam:ListInstanceProfilesForRole","iam:ListInstanceProfileTags","iam:ListMFADevices","iam:ListMFADeviceTags","iam:ListOpenIDConnectProviders","iam:ListOpenIDConnectProviderTags","iam:ListPolicies","iam:ListPoliciesGrantingServiceAccess","iam:ListPolicyTags","iam:ListPolicyVersions","iam:ListRolePolicies","iam:ListRoles","iam:ListRoleTags","iam:ListSAMLProviders","iam:ListSAMLProviderTags","iam:ListServerCertificates","iam:ListServerCertificateTags","iam:ListServiceSpecificCredentials","iam:ListSigningCertificates","iam:ListSSHPublicKeys","iam:ListVirtualMFADevices","iam:PutRolePolicy","iam:TagOpenIDConnectProvider","iam:TagPolicy","iam:TagRole","iam:UntagOpenIDConnectProvider","iam:UpdateAssumeRolePolicy","iam:UpdateRole","kms:CreateAlias","kms:CreateGrant","kms:CreateKey","kms:DeleteAlias","kms:DescribeKey","kms:EnableKeyRotation","kms:GetKeyPolicy","kms:GetKeyRotationStatus","kms:ListAliases","kms:ListResourceTags","kms:ScheduleKeyDeletion","kms:TagResource","kms:UpdateAlias","logs:CreateLogGroup","logs:CreateLogStream","logs:DeleteLogGroup","logs:DeleteMetricFilter","logs:DescribeLogGroups","logs:DescribeMetricFilters","logs:FilterLogEvents","logs:GetLogGroupFields","logs:ListTagsLogGroup","logs:PutLogEvents","logs:PutMetricFilter","logs:PutRetentionPolicy","logs:TagLogGroup","logs:UntagLogGroup","s3:CreateBucket","s3:DeleteBucket","s3:DeleteBucketPolicy","s3:DescribeJob","s3:Get*","s3:List*","s3:PutBucketAcl","s3:PutBucketOwnershipControls","s3:PutBucketPolicy","s3:PutBucketPublicAccessBlock","s3:PutBucketTagging","s3:PutBucketVersioning","s3:PutEncryptionConfiguration","s3:PutObject","servicequotas:ListServiceQuotas","sts:DecodeAuthorizationMessage"],"Effect":"Allow","Resource":"*","Sid":"BigAnimalBasePolicy"},{"Action":["ssm:AddTagsToResource","ssm:GetParameters","ssm:PutParameter","ssm:DeleteParameters"],"Effect":"Allow","Resource":"arn:aws:ssm:*:*:parameter/BA_*","Sid":"BigAnimalSystemsManagerParameterStorePolicy"},{"Action":["iam:PassRole","logs:FilterLogEvents"],"Effect":"Allow","Resource":["arn:aws:iam::*:role/ba-${external_id}-*","arn:aws:iam::*:role/biganimal-${suffix_role_name}","arn:aws:logs:*:*:log-group:/ecs/ba-bootstrap:log-stream:"],"Sid":"BigAnimalECSTaskPolicy"},{"Action":"secretsmanager:*","Effect":"Allow","Resource":"arn:aws:secretsmanager:*:*:secret:BA*","Sid":"BigAnimalSecretsPolicy"},{"Sid":"BigAnimalS3Policy","Effect":"Allow","Action":["s3:DeleteObject","s3:DeleteObjectVersion"],"Resource":"arn:aws:s3:::pg-*/*"}]}
```

Comment:
LONNNNG lines! 
can we see about prettifying that for easier reviews (sometime?)

Reply:
yes, the PP version is aws/biganimal_AWS_basepolicy.json, this is why I always update both

Reply:
is it fine like this?

Reply:
fine for now, for sure